aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -0,0 +1,230 @@
+## Installation Guide for Windows OS
+
+### A one-time setup:
+
+- Download [Python 3.6.5](https://www.python.org/downloads/release/python-365) (Windows x86-64 executable installer version 3.6.5) 
+  
+  **Note:** Don't forget to check **Add to path** while installing.
+
+- Check the python version 
+    ```
```

Comment:
can you dedent all of these code blocks? I see no reasons to indent

Reply:
U mean to say that there should be no indentation the code blocks should be just after the after eg: just below check is that what u are saying @SanketDG ?